### ISSUE TITLE
[Feature] Support for Expo plugins

### ISSUE THREAD

**Issue by Jonatthu** [mrousavy/nitro#163]
Any tutorial or template of an expo plugin to make life easier for expo users and register any necessary lines of code on Swift or Android files that the tutorials ask to write down on swift and gradle? 

<img width="731" alt="image" src="https://github.com/user-attachments/assets/1c75dc3f-4e62-4a28-9176-039f070dd391">

**Comment by mrousavy** [comment]
Good idea, but haven't build that yet. I'll gladly accept any PRs if you want to tackle this! :)

**Comment by mrousavy** [comment]
The recommended way to use Nitro is as a separate library, and there you don't need Expo Plugins - it just works. So I'm closing this
